raft/raft_test.go

```diff
@@ -599,13 +601,13 @@ func TestHandleHeartbeat2AA(t *testing.T) {
 		sm := newTestRaft(1, []uint64{1, 2}, 5, 1, storage)
 		sm.becomeFollower(2, 2)
```

Comment:
```suggestion
		sm.becomeFollower(3, 2)
```
Should be the last log's term.